site/networking.xml

```diff
@@ -0,0 +1,671 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:doc="http://www.rabbitmq.com/namespaces/ad-hoc/doc"
+      xmlns:xi="http://www.w3.org/2003/XInclude"
+      xmlns:x="http://www.rabbitmq.com/2011/extensions">
+  <head>
+    <title>Networking and RabbitMQ</title>
+  </head>
+  <body>
+    <doc:section name="intro">
+      <doc:heading>Introduction</doc:heading>
+
+      <doc:subsection name="intro">
+        <p>
+          Clients communicate with RabbitMQ over the network. All
+          protocols supported by the broker are TCP-based. As such,
+          both RabbitMQ and the operating system provide a wide range
+          of knobs that can be tweaked. Some of them are directly
+          related to TCP and IP operations, others have to do with
+          application-level protocols such as TLS. This guide covers
+          multiple topics related to networking in the context of
+          RabbitMQ. This guide is not meant to be an extensive
+          reference but rather an overview. Some tuneable parameters
+          discussed are OS-specific. This guide focuses on Linux when
+          covering OS-specific subjects, as it is the most common
+          platform RabbitMQ is deployed on in production.
+        </p>
+        <p>
+          There are several areas which can be configured or tuned:
+
+          <ul>
+            <li>Interfaces and ports</li>
+            <li>TLS</li>
+            <li>TCP socket settings</li>
+            <li>Kernel TCP settings</li>
+            <li>(AMQP 0-9-1, STOMP) Heartbeats</li>
+            <li>Hostnames and DNS</li>
+          </ul>
+
+          Except for OS kernel parameters and DNS, all RabbitMQ settings
+          are <a href="/configure.html">configured the same way</a>.
+        </p>
+        <p>
+          Networking is a broad topic. There are many configuration options
+          that can have positive or negative effect on certain workloads.
+          As such, this guide does not try to be a complete reference but rather
+          offer an index of key tunable parameters and serve as a starting
+          point.
+        </p>
+      </doc:subsection>
+    </doc:section>
+
+    <doc:section name="interfaces">
+      <doc:heading>Network Interfaces</doc:heading>
+
+      <doc:subsection name="multiple-interfaces">
+        <p>
+          For RabbitMQ to accept client connections, it needs to bind to one or more
+          interfaces and listen on (protocol-specific) ports. The interfaces are
+          configured using the <code>rabbit.tcp_listeners</code> config option.
+          By default, RabbitMQ will listen on all available interfaces.
```

Comment:
Perhaps change to "By default, RabbitMQ will listen on port 5672 on all available interfaces."